src/components/Login.js

```diff
@@ -14,22 +25,29 @@ const Login = () => (
           <FormLabel labelStyle={commonStyle.noSideMargins}>
             用户名或邮箱
           </FormLabel>
-          <FormInput containerStyle={commonStyle.noSideMargins} />
+          <FormInput
+            containerStyle={commonStyle.noSideMargins}
+            onChangeText={setLoginName}
+          />
           <FormLabel labelStyle={commonStyle.noSideMargins}>密码</FormLabel>
           <FormInput
             containerStyle={commonStyle.noSideMargins}
+            onChangeText={setPasswd}
             secureTextEntry
           />
+          <FormValidationMessage>{errorMessage}</FormValidationMessage>
```

Comment:
1. Add `labelStyle={commonStyle.noSideMargins}` to make it look better.
2. The warning message remains showing even after the user have successfully logged in and signed out. Consider remove the warning message when
    - the user switches to another page. In this case, the input boxes should also be cleared as well.
    - the user changes the content of the input boxes.
    - the user logs in successfully.